Project_FreeRTOS/main/task_rung.h

```diff
@@ -6,56 +6,50 @@
 
 void task_RUNG(void *pvParameters)
 {
-    const int rung_noiseThreshold = ....; // cập nhật giá trị
-    int rung_unchangedCount=0;
-    int rung_lastValue1 = 0;
-    int rung_lastValue2 = 0;
-    int rung_count=0;
-    unsigned long rung_lastSignalTime = 0;
+    int rung_noiseThreshold = 50; // cập nhật giá trị
+    int rung_lastValue = 0;
+    int rung_count = 0;
   for(;;)
   {
-    unsigned long rung_currentTime = millis(); // đọc giá trị thời gian hiện tại
-
-
-    int rung_value = analogRead(IN_SW_1801P);
+    int rung_currentvalue = analogRead(IN_SW_1801P);
     Serial.print("Rung Value: ");
-    Serial.println(rung_value); 
+    Serial.println(rung_currentvalue); 
    Message sw_send = {0};
       sw_send.id_Tx = OUT_BUZZER;
       sw_send.id_Rx = IN_SW_1801P;
       sw_send.payload = BUZZER_SW_DISABLE;
```

Comment:
Please swap values of 2 lines 18 and 19
follow me:
sw_send.id_Tx = IN_SW_1801P;
sw_send.id_Rx = OUT_BUZZER;